docs/user-guide/annotations.md

```diff
@@ -26,3 +26,12 @@ resource "aws_instance" "foo" {
     instance_type = "t1.2xlarge"
 }
 ```
+
+It is possible to add a reason for why a rule is ignored:
+
+```hcl
+resource "aws_instance" "foo" {
+    # tflint-ignore: aws_instance_invalid_type # A good reason
+    instance_type = "t1.2xlarge"
+}
+```
```

Comment:
When placing the annotation on the previous line, also breaking the comment onto its own line(s) seems preferable. But this is a nice addition for users of same-line annotations, or very short comments.

GitHub PR suggestions don't allow for back-ticked blocks inside, here's what I have in mind:


<hr>

It is possible to add a reason for why a rule is ignored:

```hcl
resource "aws_instance" "foo" {
    # This instance type is new and TFLint doesn't know about it yet
    # tflint-ignore: aws_instance_invalid_type
    instance_type = "t10.2xlarge"
}
```

Or, on the same line:

```hcl
resource "aws_instance" "foo" {
    instance_type = "t10.2xlarge" # tflint-ignore: aws_instance_invalid_type # too new for TFLint
}
```

Reply:
> GitHub PR suggestions don't allow for back-ticked blocks inside, here's what I have in mind:

It actually does work! (Thanks @FluffyBucket for showing me!) You just need to use more than 3 backticks on the outermost block.


````suggestion
It is possible to add a reason for why a rule is ignored:

```terraform
resource "aws_instance" "foo" {
    # This instance type is new and TFLint doesn't know about it yet
    # tflint-ignore: aws_instance_invalid_type
    instance_type = "t10.2xlarge"
}
```

Or, on the same line:

```terraform
resource "aws_instance" "foo" {
    instance_type = "t10.2xlarge" # tflint-ignore: aws_instance_invalid_type # too new for TFLint
}
```
````

I'll add this, together with @wata727's suggestion about `//`